modules/core/src/main/java/org/apache/ignite/internal/processors/cache/checker/tasks/RepairRequestTask.java

```diff
@@ -93,26 +93,27 @@
         Map<UUID, Map<KeyCacheObject, Map<UUID, VersionedValue>>> targetNodesToData = new HashMap<>();
 
         for (Map.Entry<KeyCacheObject, Map<UUID, VersionedValue>> dataEntry: repairReq.data().entrySet()) {
-            KeyCacheObject keyCacheObject;
+            KeyCacheObject keyCacheObj;
+
+            GridCacheContext<Object, Object> ctx = ignite.cachex(repairReq.cacheName()).context();
 
             try {
-                keyCacheObject = unmarshalKey(dataEntry.getKey(), ignite.cachex(repairReq.cacheName()).context());
+                keyCacheObj = unmarshalKey(dataEntry.getKey(), ctx);
             }
-            catch (IgniteCheckedException e) {
+            catch (IgniteCheckedException ignored) {
```

Comment:
Seems that it's not ignored cause we log an exception.

Reply:
We didn't log an exception; fixed that and variable name.